pilot/pkg/bootstrap/server.go

```diff
@@ -761,6 +762,8 @@ func (s *Server) initSecureDiscoveryService(args *PilotArgs) error {
 	s.XDSServer.Register(s.secureGrpcServer)
 	reflection.Register(s.secureGrpcServer)
 
+	admin.Register(s.secureGrpcServer)
```

Comment:
I am only seeing metrics for IstioCertificateService for some reason, not sure if that is expected?

Reply:
And on agent not seeing any either. Does it only support unary?

its possibly a good thing to not expose on the agent.. we have concerns about metrics cardinality at agent

Reply:
Oh I think it lazy loads the metrics, and it only registers after stream close. So when once I had disconnected once they did show up.

This adds 256 new metrics to the agent. I think its way too much, and not *that* useful - envoy itself is only adding 286 and it already causes complaints.

Reply:
I'll add an env variable to control. 

Like Istio, gRPC has some 'standard' metrics and debug tools - and it seems reasonable to support them, but agree we may not want them in all cases.

For proxyless - there is no envoy, and we are mostly blind ( metrics-wise ).